packages/web-components/src/divider/divider.styles.ts

```diff
@@ -125,4 +125,22 @@ export const styles = css`
   :host([appearance='subtle']) ::slotted(*) {
     color: ${colorNeutralForeground3};
   }
-`;
+`.withBehaviors(
+  forcedColorsStylesheetBehavior(css`
+    /* high contrast styles */
```

Comment:
No need to add this comment or space, it's implied by the context of the behavior